packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initEditUserModalMixin.ts

```diff
@@ -89,6 +104,12 @@ export const initEditUserModalMixin = createSingletonMixin(
           email: userDetails?.email,
           phone: formatPhoneNumber(userDetails?.phone),
           roles: userDetails?.roles,
+          ...Object.fromEntries(
+            Object.entries(userDetails.customAttributes).map(([key, val]) => [
+              `customAttributes.${key}`,
+              val,
+            ]),
+          ),
```

Comment:
Let's extract it into a `flatten` function

Reply:
Dome